ultravox/training/configs/stage2_lora.yaml

```diff
@@ -1,16 +1,23 @@
+exp_name: stage2_lora__gs_ai_bq
+
 text_model_lora_config:
   r: 64  # no/little change in the range [16, 64]
   target_modules: ['mlp.gate_proj', 'mlp.up_proj', 'mlp.down_proj', 'v_proj', 'o_proj', 'k_proj', 'q_proj']
 
-data_sets: ["commonvoice", "peoplespeech", "anyinstruct"]
+data_sets: ["gigaspeech", "anyinstruct", "boolq_extended"]
 
 # disable_layer_drop: True
 # audio_model_lora_config:
 #   r: 64
 #   target_modules: ['k_proj', 'q_proj', 'v_proj', 'out_proj', 'intermediate_dense', 'output_dense']
 
-num_prompts: 6
+num_prompts: 11
 
 lr: 1.e-4  # need a lower LR for LLM fine-tuning
+lr_scheduler: constant_with_warmup
 lr_warmup_steps: 250
 max_steps: 5_000
+save_steps: 0
+
+batch_size: 2
+grad_accum_steps: 2
```

Comment:
When calculating the budget for SODA I noticed that some BoolQ passages are very long. It's possible that we can filter those out and keep `batch_size=4`.

TODO.

Reply:
IIUC, with Llama3 and Whisper, our token budget with `batch_size=4` is somewhere between 668 and 728.

We could definitely filter BoolQ to get rid of the few samples that fall in this regime (less than 1%).

With SODA, the ratio is higher though (depending on where we draw the line).

Reply:
Okay nevermind. The training did not get faster.
GPU utilization was already at > 90% and returning batch_size to 4 even somehow made things slower. Reverting to `batch_size=2 grad_acc_steps=2`.

Reply:
Marking PR ready to review.